### ISSUE TITLE
NWS Watches and Warnings Layer

### ISSUE THREAD

**Issue by wilsaj** [TNRIS/flood#48]
A map layer containing active Watches and Warnings from the National Weather Service

## source
- available through Aeris weather, which updates every 2-3 minutes
- original source of data is NWS: http://www.weather.gov/ 
- have a rough pass at a five-minute updating geojson layer at http://labs.tnris.org/alerts.geojson

## status
- [ ] waiting on procurement of Aeris subscription #31

**Comment by wilsaj** [comment]
closing, duplicate of #25
